README.md

```diff
@@ -158,37 +158,42 @@ Option | Type | Default | Description
 
 <a name="tagsOption"></a>
 ##### tags (optional, defaults to `[]`)
-An array of tags that are displayed as pre-selected. Each tag should have an  `id` and `text` property which is used to display.
+An array of tags that are displayed as pre-selected. Each tag should have an `id` property and a property for the label, which is specified by the [`labelField`](#labelFieldOption).
 
 ```js
+// With default labelField
 const tags =  [ { id: "1", text: "Apples" } ]
+
+// With labelField of `name`
+const tags =  [ { id: "1", name: "Apples" } ]
 ```
 
 <a name="suggestionsOption"></a>
 ##### suggestions (optional, defaults to `[]`)
-An array of suggestions that are used as basis for showing suggestions. At the moment, this should be an array of strings.
+An array of suggestions that are used as basis for showing suggestions. These objects should follow the same structure as the data. So if the `labelField` is `name`, the following would work:
```

Comment:
These objects should follow the same structure as the  [`tags`](#tagsOption).
So if the `labelField` is `name`, the following would work: